maven-failsafe-plugin/pom.xml

```diff
@@ -45,6 +45,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Is there a plexus logging dependency we can remove yet?

Reply:
I wasn't sure with an initial glance, will look more deeply now.

Reply:
In this repo's deps, found `org.codehaus.plexus.logging.Logger` exists in:
* plexus-component-api-1.0-alpha-20
* plexus-container-default-1.0-alpha-9-stable-1
* org.eclipse.sisu.plexus-0.9.0.M3

Those deps are all transient thru multiple deps and appear still used.  I don't see any we can currently remove, but maybe there are.  Is there anything else I should investigate with dep removal?

Reply:
The transitive deps don't need to be declared in this pom.xml, only the direct dependencies. plexus-component-api-1.0-alpha-20 and plexus-container-default-1.0-alpha-9-stable-1 need to go away too though again not in this PR.